### ISSUE TITLE
%arguments is not available in 2.3

### ISSUE THREAD

**Issue by gsmeets** [canjs/canjs#2988]
I'd like to see this feature from 3.0 in 2.3 as well, for us that are stuck on 2.3 for the time being. 
Will you accept a pull request for this feature?

**Comment by phillipskevin** [comment]
Yes, we will accept this if you submit a PR against the `2.3-legacy` branch. Here is the PR that added it to can-stache-bindings: https://github.com/canjs/can-stache-bindings/pull/66.

**Comment by phillipskevin** [comment]
@gsmeets sorry, I originally said we would get this in, but since this would require a 2.4 release, maybe @justinbmeyer should weigh in.

**Comment by justinbmeyer** [comment]
I'm ok adding this in w/i a 2.3 release.  We'd violate semver, but not the way that really matters to people IMO.  

Releasing a 2.4 is much more complex b/c we need to build a different site for it.  Doesn't seem worth it.

**Comment by phillipskevin** [comment]
@gsmeets since I deleted my original response, here is where this was added in 3.0: https://github.com/canjs/can-stache-bindings/blob/master/can-stache-bindings.js#L291.

You should just need to add that here: https://github.com/canjs/canjs/blob/2.3-legacy/view/bindings/bindings.js#L264-L279 and copy this test over https://github.com/canjs/can-stache-bindings/blob/master/test/bindings-test.js#L2416-L2429.

**Comment by justinbmeyer** [comment]
@phillipskevin on second thought, maybe we release it as 2.4, but just don't publish docs for it?

**Comment by phillipskevin** [comment]
Why not just publish 2.4 docs to v2.canjs.com?

**Comment by justinbmeyer** [comment]
sure.  It just always seems error prone.

**Comment by phillipskevin** [comment]
Ok, we can decide when the time comes. I'm fine either way.

**Comment by gsmeets** [comment]
yeah, I already found the line and have it hotfixed in my project. One thing I noticed that was a bit odd btw was that the event is part of those arguments. Didn't really expect that, as you can pass that in seperately if needed. (In fact, I would've expected the argument list to be flattened into whatever else you'd pass in, but I guess that can cause its own problems for people.)
